.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: Pull request
+'on':
+  pull_request:
+jobs:
+  tox:
+      runs-on: ubuntu-20.04
+      strategy:
+        matrix:
+          include:
+            - environment: pep8
+              python-version: 3.6 
```

Comment:
why not 22.04 and python3.10? It doesn't make sense to use EOL python in a CI job that only generates renos, does it?